RockLib.Configuration/Config.cs

```diff
@@ -90,12 +89,16 @@ public static void SetRoot(Func<IConfiguration> getRoot)
         /// <param name="additionalValues">When specified, these key/value pairs are applied to the resulting
         /// instance of <see cref="IConfiguration"/>.</param>
         /// <exception cref="InvalidOperationException">If the <see cref="IsLocked"/> property is true.</exception>
```

Comment:
Do we need to add any exception references here because you removed the suppression of exceptions in AddRockLibSecrets?

Reply:
Hmmmm....this `<exception>` tag has been there for a while, and I'm not sure why it's specific for just that one type. I'll remove it.

Reply:
Thats fine if its not thrown anymore.  But I was just wondering if there will be issues now since we removed the suppression of all exceptions in AddRockLibSecrets.  I was thinking we could at least put the documentation there so that there are no surprises for the user.